### PR TITLE
SPCTR-3148 : [Core] Remove old post assets

### PR DESCRIPTION
**Pull request by adi3890**
### Description
This was reported that new file assets where generated whenever editor was updated.
This solution cheks if already file exists and deltes it, before generating new one.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: classes/class-uagb-post-assets.php [hunk 1]
```diff
@@ -1174,6 +1174,16 @@ public function create_file( $file_data, $type, $file_state = 'new', $old_file_n
 
 		$result = false;
 
+		// Remove if any old file exists for same post.
+		$old_assets = glob( $base_file_path . 'uag-' . $type . '-' . $this->post_id . '*' );
```
Comment (AbhijitNage123) [inline classes/class-uagb-post-assets.php:1178]: @adi3890 Any reason behind to appended an `astric *`.

Comment (adi3890) [reply classes/class-uagb-post-assets.php]: regex whatever timestamp comes further we will pick all.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -216,17 +216,18 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Improvement: Global - Removing unneccessary archieved assets.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Improvement: Global - Added auto-cleaning for archieved assets.
```

**Review by bsf-zanev — APPROVED** [review]
Just needs changelog update.

File: README.md (hunk 2 above)
Comment (adi3890) [reply README.md]: updated as suggested

**Pull request merged**